examples/singlefile/README.md

```diff
@@ -1,14 +1,13 @@
 # An single-file example RESTful API app showcasing the power of pyramid_openapi3
 
-The `app.py` file in this folder showcases how to use the [pyramid_openapi3](https://github.com/niteoweb/pyramid_openapi3) Pyramid add-on for building robust RESTful APIs. With only a few lines of code you get automatic validation of requests and responses against an OpenAPI v3 schema, along with Swagger "try-it-out" documentation for your API.
+The `app.py` file in this folder showcases how to use the [pyramid_openapi3](https://github.com/Pylons/pyramid_openapi3) Pyramid add-on for building robust RESTful APIs. With only a few lines of code you get automatic validation of requests and responses against an OpenAPI v3 schema, along with Swagger "try-it-out" documentation for your API.
 
 ## How to run
 
 ```bash
-* git clone https://github.com/niteoweb/pyramid_openapi3.git
-* cd pyramid_openapi3/examples/singlefile
-* virtualenv -p python3.7 .
-* source bin/activate
+* mkdir my-openapi
+* cd my-openapi
+* wget https://raw.githubusercontent.com/Pylons/pyramid_openapi3/master/examples/singlefile/app.py
```

Comment:
`wget` is unusual to Pylon Project projects. In every project I have examined, we follow the practice of `git clone`, and create a virtual environment (this should _always_ be done anyway).

Most Pylons Project projects avoid activating the virtual environment and suggest not doing so (see note under https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/install.html#installing-pyramid-on-a-unix-system). We have lots of experience supporting newbs, and Pyramid's path is the one of least pain and suffering.

Of course, if you know what you're doing, then go crazy. Just don't expect a newb to know WTF they are doing when they blindly copy-pasta commands.

Reply:
But why would I need to clone the repository if I want to see how pyramid_openapi3 works? Do you need to clone e.g. django rest framework if you want to see how it works? Ref https://www.django-rest-framework.org/tutorial/quickstart/
In Flask api you also don't need to clone their repo to take it for spin. Ref https://www.flaskapi.org/#example

That said I also don't like the "wget" part but I couldn't think of a better solution.

Reply:
There's a demo URL to see how it works. Your ref links either ignore the creation of a virtual environment, which is a user-hostile thing to do, or lack sufficient detail to develop beyond an example.

I think the context here is less "how does it work", but more "how do I start using it in my project" or "how can I contribute to it", for which `git clone` is more appropriate.

Alternatively, perhaps a cookiecutter would be a better option for a quick start. [We use it for Pyramid](https://github.com/pylons/pyramid-cookiecutter-starter).